pkgs/misc/vencord/default.nix

```diff
@@ -36,13 +36,15 @@ buildNpmPackage rec {
   # Supresses an error about esbuild's version.
   npmRebuildFlags = [ "|| true" ];
 
-  npmDepsHash = "sha256-GoVVOLg20oi0MJGLqevpiqHDM/7yaRJSQnM/tt+AkQ8=";
+  makeCacheWritable = true;
```

Comment:
Why is this needed?

Reply:
otherwise it has the issues with writing to cache, basically.

Reply:
```
npm ERR! code EACCES
npm ERR! syscall mkdir
npm ERR! path /nix/store/qlps29dh4gavnccma4k8sa61jhqq36py-vencord-1.5.3-npm-deps/_cacache/tmp
npm ERR! errno -13
npm ERR! 
npm ERR! Your cache folder contains root-owned files, due to a bug in
npm ERR! previous versions of npm which has since been addressed.
npm ERR! 
npm ERR! To permanently fix this problem, please run:
npm ERR!   sudo chown -R 1000:100 "/nix/store/qlps29dh4gavnccma4k8sa61jhqq36py-vencord-1.5.3-npm-deps"
```

Reply:
I am wondering why this is needed now. How did you generate your updated package-lock.json?

Reply:
through `npm i` on `tags/v1.5.3` of `Vendicated/Vencord`

Reply:
Could you try regenerating it using `npm install --legacy-peer-deps -f`? That's what @FlafyDev and I have been doing IIRC

Reply:
> Could you try regenerating it using `npm install --legacy-peer-deps -f`? That's what @FlafyDev and I have been doing IIRC

ah, my bad, `--legacy-peer-deps` was used, too. didn't use `-f` tho